docs/guides/build/linux.md

```diff
@@ -0,0 +1,3 @@
+# Build Linux
+
+todo
```

Comment:
maybe we need a tracking issue here assigned to @steverusso

Reply:
or you can just chat with him and then do this.

Reply:
I'm in favor of having him commit to this PR directly